lib/App/DH.pm

```diff
@@ -281,6 +317,8 @@ my $list_cmds_usage =
     cmd_write_ddl
     cmd_install
     cmd_upgrade
+    cnd_database_version
```

Comment:
#### Squashing
Yeah. Its really a case-by-case thing. The individual commits themselves should be logical and atomic, and passing tests.

Squashing can be useful at this, but squashing is just a tool, and you have to use the right tool for the right job :)

Squashing unrelated changes for instance is obviously bad.

#### PRs.

No, I don't appear to get more emails for pushes. I thought it did, but I can't find any evidence it does.

So the only thing I really notice is when coveralls adds its bit after a travis run.

( But I don't really mind when people submit things that affect coverage, its not a hard metric, and I wish there was a way to disable the implication that reduced coverage is a failure ... )